sample-bootstrap.sh

```diff
@@ -15,4 +15,4 @@ echo 'EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_simulator__NATIVE_ARCH_64_BIT_x8
 echo 'EXCLUDED_ARCHS = $(inherited) $(EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_$(EFFECTIVE_PLATFORM_SUFFIX)__NATIVE_ARCH_64_BIT_$(NATIVE_ARCH_64_BIT)__XCODE_$(XCODE_VERSION_MAJOR))' >> $xcconfig
 
 export XCODE_XCCONFIG_FILE="$xcconfig"
-carthage bootstrap
+carthage bootstrap --platform iOS
```

Comment:
https://getstream.slack.com/archives/G019728A97W/p1601387865020100?thread_ts=1601376554.016900&cid=G019728A97W

Reply:
Yep I added it after the comment :D

Reply:
?? But it shouldn't be there, the dependencies are used by both v2 and v3 samples